### PR TITLE
Working version of Arbitrable abstract contract

### PR DESCRIPTION
**Pull request by clesaege**
+ small arbitrable contract which only modify its state

### CODE REVIEW

File: decentralized-court/contracts/Arbitrated/Arbitrated.sol [hunk 1]
```diff
@@ -147,13 +148,13 @@ contract TwoPartyArbitrable is Arbitrable {
     
     /** Set the state in the absence of reaction of the other party.
      *  @param executeA True if A to be executed. False if B to be executed.
-     *  TO DO: Verify that we can't abuse of with the appeal mechanism.
```
Comment (n1c01a5) [inline decentralized-court/contracts/Arbitrated/Arbitrated.sol:150]: maybe `TODO`

File: decentralized-court/contracts/Arbitrated/Arbitrated.sol [hunk 2]
```diff
@@ -74,7 +75,7 @@ contract TwoPartyArbitrable is Arbitrable {
     function ruleB(uint256 disputeID) onlyCourt { executeRulingB(disputeID); }
     
     /// Set the state at ruledA. The court call this function if A is ruled.
-    function executeRulingA(uint256 _disputeID) private{
+    function executeRulingA(uint256 _disputeID) private {
         if (_disputeID!=disputeID) // There were no dispute
```
Comment (n1c01a5) [inline decentralized-court/contracts/Arbitrated/Arbitrated.sol:79]: Add `.` to finish the comment sentence.

**Review by n1c01a5 — CHANGES_REQUESTED** [review]

File: decentralized-court/contracts/Arbitrated/Arbitrated.sol (hunk 2 above)
Comment (clesaege) [reply decentralized-court/contracts/Arbitrated/Arbitrated.sol:79]: Change to "Not arbitrating the current dispute.".

File: decentralized-court/contracts/Arbitrated/Arbitrated.sol (hunk 1 above)
Comment (clesaege) [reply decentralized-court/contracts/Arbitrated/Arbitrated.sol:150]: OK

**Pull request closed without merging**